renv/core.py

```diff
@@ -132,49 +131,65 @@ def ensure_directories(self, env_dir):
         r_script = "Rscript"
         context.R_exe = r_exe
         context.R_script = r_script
-        context.R_version = os.path.split(self.r_path)[1]
         if self.r_bin_path:
             context.abs_R_exe = os.path.join(self.r_bin_path, r_exe)
             context.abs_R_script = os.path.join(self.r_bin_path, r_script)
         else:
             context.abs_R_exe = os.path.join(self.r_path, "bin", r_exe)
             context.abs_R_script = os.path.join(self.r_path, "bin", r_script)
         context.abs_R_path = self.r_path
-        logging.info(f"System R(version):  {self.r_path}({context.R_version})")
 
+        # Get the version of R
+        # Major Version
+        major, error = utils.system_r_call(rcmd_type="major", context=context)
+        # Minor Version
+        minor, error = utils.system_r_call(rcmd_type="major", context=context)
+
+        context.R_version = "%s.%s" % (major, minor)
```

Comment:
What if we actually got an error in the system_r_call? i.e. stdout is not actually the information we want. We should handle that.

Reply:
Will handle in future updates #49